test/fixtures/test_logger.js

```diff
@@ -19,6 +19,13 @@ class TestLogger {
     return true;
   }
 
+  clear() {
```

Comment:
I think we don't need to clear it because it comes with empty logs when being newed.